components/odh-notebook-controller/e2e/notebook_update_test.go

```diff
@@ -65,7 +65,9 @@ func (tc *testContext) testNotebookUpdate(nbContext notebookContext) error {
 	}
 
 	// Example update: Change the Notebook image
-	updatedNotebook.Spec.Template.Spec.Containers[0].Image = "new-image:latest"
+	// newImage := "new-image:latest"  quay.io/thoth-station/s2i-minimal-notebook:v0.2.2
+	newImage := "quay.io/opendatahub/workbench-images:jupyter-minimal-ubi9-python-3.11-20241119-3ceb400"
```

Comment:
Question here: I deliberately used our own workbench repository for this. But for the original first workbench, there is used that `quay.io/thoth-station/s2i-minimal-notebook`... soo... let me know if we want that instead for some reason :shrug:

Reply:
I put toth into the github actions when I was touching these things. I guess it does not really matter, so let's go with your choice.

Reply:
the images must be consistent with whatever the readiness / and-the-other-one probes that are defined on notebook CR, guess that's pretty much it and anything else would work; maybe that the entrypoint runs indefinitely, that may be also important, that the image does not exit on its own (or does not exit quickly)